docs/ism/policies.md

```diff
@@ -321,6 +322,26 @@ Parameter | Description | Type | Required | Default
 }
 ```
 
+### index_priority
+
+Set the priority for the index in a specific state. Unallocated shards of indices are recovered in the order of their priority, whenever possible.
```

Comment:
By default, what priority (if any) do indices have? For priority, would 1 be extremely high or extremely low?

Reply:
Added the default priority and some context. Please review.